README.md

```diff
@@ -1,130 +1,61 @@
---- Getting the data from the DB as part of your notebook means you can get updated data just by re-running the notebook. However, since the database table is subject to change, it's also smart to save off the data in a file alongside the notebook so that you can run the notebook later without requiring database access.
---- Recommended practice is to use the database code while the experiment is in progress or when doing a single data pull, but always save that data off to a CSV and switch to using the CSV before you archive the notebook.
-
-- Technician Access to Calculation Details
--- Pulling calculation details from the DB
---- The calculation details table has all kinds of handy internal intermediate details for each calculation.
---- Getting the data from the DB as part of your notebook means you can get updated data just by re-running the notebook. However, since the database table is subject to change, it's also smart to save off the data in a file alongside the notebook so that you can run the notebook later without requiring database access.
---- Recommended practice is to use the database code while the experiment is in progress or when doing a single data pull, but always save that data off to a CSV and switch to using the CSV before you archive the notebook.
-
-# End Jason Notebook
+# Jupyter Notebook Tools
+This is a set of tools that is designed to facilitate science in the context of Osmo Systems.
+The code herein generally expects to be run in the context of a jupyter notebook. TODO (PR feedback plz): eventually it might be used elsewhere. is it OK to put it all here for now and potentially have to refactor/move it later?
+
+## What lives here
+
+Here's the basic heuristic for what should live in this library vs. in a notebook.
+
+**Is this function reusable across many experiments/activities, both now and for the foreseeable future, and is it fairly Osmo-specific?**
+yes = probably it should live here. no = probably not.
+
+Examples of code that should live here:
+
+* Generalized database access functions
+* Basic plots that are useful for a wide variety of experiments
+* Calibration routines
+* Analysis functions
+
+Examples of code that should probably *not* live here:
+
+* "get the standard deviation of a set of node data". Use `my_dataframe['my_column'].std()` as [implemented in pandas](https://pandas.pydata.org/pandas-docs/stable/generated/pandas.Series.std.html) instead.
+* "check temperature and time correlation of each color count for each node during an experiment". This is probably too use-case specific and will end up as unmaintained, "dead code".
+
+## Using this library
+
+To consume this library in your notebook, add this line to the top:
+
+`!pip install https://github.com/osmosystems/python-jupyter-tools.git@[VERSION]`
+Where [VERSION] is a version number.
+
+TODO: instructions for installing editable for testing/development purposes. include autoreload magics
+
+## Contributing
```

Comment:
Do we need a ## Versioning subsection as well?

Reply:
what would you put under that?

Reply:
I was thinking something akin to a ```CHANGELOG.MD```.

Reply:
ah, right. I was gonna leave that out for now - since we're not doing semver, the utility is a bit less and basically you should just use the git commit log.